cobigen/cobigen-core/src/test/java/com/devonfw/cobigen/unittest/adapter/TemplateAdapterTest.java

```diff
@@ -0,0 +1,84 @@
+package com.devonfw.cobigen.unittest.adapter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.devonfw.cobigen.api.TemplateAdapter;
+import com.devonfw.cobigen.api.constants.ConfigurationConstants;
+import com.devonfw.cobigen.impl.adapter.TemplateAdapterImpl;
+
+/**
+ * Tests for {@link TemplateAdapterImpl}
+ */
+public class TemplateAdapterTest {
+
+  /**
+   * Root Path for tests with the monolithic template structure
+   */
+  private static final Path rootTestPathMonolithicTemplates = new File(
+      "src/test/resources/testdata/unittest/TemplateAdapterTest/homeMonolithicTemplates/templates").toPath();
+
+  /**
+   * Root Path for tests with the template set structure
+   */
+  private static final Path rootTestPathTemplateSets = new File(
+      "src/test/resources/testdata/unittest/TemplateAdapterTest/homeTemplateSets/template-sets").toPath();
```

Comment:
Please use: ConfigurationConstants.TEMPLATE_SETS_FOLDER instead of template-sets.